eng/pipelines/test-windows-job.yml

```diff
@@ -31,6 +31,11 @@ jobs:
   steps:
     - checkout: none
 
+    - task: UseDotNet@2
```

Comment:
The tests aren't being run on this machine. They are sent and run on helix (correct me if I am wrong @RikkiGibson). This machine just waits for the helix results to report as passed (however if the single-machine run passes that will tell us what the problem is).

Reply:
doh!

Reply:
Although should at least see the single machine test pass...

Reply:
@jmarolf And Windows_CoreClr_Single_Machine did pass... hmm.. I could just the windows coreclr tests to not run on helix.